source/borderrouter_thread_tasklet.c

```diff
@@ -451,6 +451,19 @@ static int backhaul_interface_down(void)
     return retval;
 }
 
+#ifdef MBED_CONF_APP_DEBUG_TRACE
+#if MBED_CONF_APP_DEBUG_TRACE == 1
```

Comment:
Why not just
```
#if MBED_CONF_APP_DEBUG_TRACE
```

Because: http://c0x.coding-guidelines.com/6.10.1.html
`1878 After all replacements due to macro expansion and the defined unary operator have been performed, all remaining identifiers (including those lexically identical to keywords) are replaced with the pp-number 0, and then each preprocessing token is converted into a token.`